tests/WorkOS/SSOTest.php

```diff
@@ -158,6 +159,7 @@ private function profileFixture()
             "email" => "hen@papagenos.com",
             "firstName" => "hen",
             "lastName" => "cha",
+            "connection_id" => "conn_01EMH8WAK20T42N2NBMNBCYHAG",
```

Comment:
```suggestion
            "connectionId" => "conn_01EMH8WAK20T42N2NBMNBCYHAG",
```

This is the resourcify'd object, gotta camel case it